### PR TITLE
Firebase for Serverless Backends and Cloud Functions

### PR DESCRIPTION
**Pull request by whyismynamerudy**
Title. 
Added section on using Firebase for Serverless backends and cloud functions. 
Please provide constructive feedback, thanks!

### CODE REVIEW

**Comment by NicholasMacasaet** [comment]
Hello Rudy, your post is clean and conscience in what topics it covers, but a way to deepen the reader's understanding would be to add some additional resources for the reader to look through and some use cases demonstrating specific functionality.  For example, you mentioned custom logic when designing serverless backends so you could add in an example to demonstrate this. 

Great work so far.

File: Topics/Tech_Stacks/FirebaseServerlessCloud.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Serverless Backends
+
+Serverless backends in Firebase encompass the entire infrastructure required to power an application, offering a range of services such as databases (Firestore, Realtime Database), authentication, hosting, and more. These backends abstract away server management, allowing developers to focus solely on writing code without worrying about underlying infrastructure concerns.
+
+## Cloud Functions
+
+Cloud Functions in Firebase are integral to implementing serverless backends within the Firebase ecosystem. They are small, event-driven functions that respond to various triggers like database events, HTTP requests, authentication events, and more. These functions execute specific tasks or logic in response to these events, providing a scalable and managed way to handle functionalities within the serverless architecture.
+
+### Integration within Firebase
```
Comment (NicholasMacasaet) [inline Topics/Tech_Stacks/FirebaseServerlessCloud.md:11]: As I mentioned in my previous comment, potentially adding some use cases to demonstrate using the cloud functions in code would help readers understand

File: Topics/Tech_Stacks/FirebaseServerlessCloud.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    try:
+        original = event.data.get("original")
+    except KeyError:
+        # No "original" field, so do nothing.
+        return
+
+    # Set the "uppercase" field.
+    print(f"Uppercasing {event.params['pushId']}: {original}")
+    upper = original.upper()
+    event.data.reference.update({"uppercase": upper})
```
Comment (NicholasMacasaet) [inline Topics/Tech_Stacks/FirebaseServerlessCloud.md:70]: after this you could add additional resources for readers to look at

File: Topics/Tech_Stacks/FirebaseServerlessCloud.md (hunk 1 above)
Comment (whyismynamerudy) [reply Topics/Tech_Stacks/FirebaseServerlessCloud.md:11]: Hey, thanks for the feedback! I added an example use case in my latest commit - please check it out when possible, thanks :)

**Review by NicholasMacasaet — APPROVED** [review]
Thanks for adding resources, this should be sufficient for a reader's further exploration into this topic. The example should be enough as well to act as a baseline for learning

**Pull request merged**